docs/tutorial/cpp/src/getting_started.md

```diff
@@ -64,13 +67,7 @@ If you are on Linux or macOS, you can run the program:
 
 and a window will appear with the green "Hello World" greeting.
 
-If you are stepping through this tutorial on a Windows machine, you need to add the `bin` sub-directory into your `%PATH%`:
-
-```sh
-set PATH=%CD%\bin;%PATH%
-```
-
-so that Windows to can find the Slint run-time library. Then you can run it with
+If you are stepping through this tutorial on a Windows machine, you can run it with
```

Comment:
Doesn't the command for linux/mac also work on windows?

Reply:
The command on linux/mac is `./memory_game`, which does not work on Windows. You'll get something about `'.'` not being recognised as a command. On the other hand, on Linux and macOS the leading `./` is required since the current directory is not in the PATH.